app/MICKAL/MICKAL_task3.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * Responsible: Michal Kaliszczyk
+ *
+ *
+ * History log:
+ * 2017.12.05   MICKAL  Initial version
+ */
+
+ 
+ #include "MICKAL_task3.h"
+ #include "stdint.h"
+ 
+ 
+static uint8_t srgu8Storage[(LAMP_COUNT % 8 == 0) ? (LAMP_COUNT/8) : (LAMP_COUNT/8)+1];
```

Comment:
1. `au8Storage` is enough (array of uin8) - btw. I never saw `srg` hungarian notation in project - ifyou have source plaese show me some so we can discuss it. Anyway - shorter is better, more human readable